md/tx_intf.md

```diff
@@ -0,0 +1,34 @@
+| Name                          | Signed? | Packed Dim             | Unpacked Dim | Clock Domain | JTAG Dir | Reset Val          |
```

Comment:
This new set of JTAG registers is basically the PI-specific subset of registers from the analog core debug interface.